include/seqan3/search/configuration/max_error.hpp

```diff
@@ -9,125 +9,90 @@
  * \brief Provides the configuration for maximum number of errors for all error types.
  * \author Christopher Pockrandt <christopher.pockrandt AT fu-berlin.de>
  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \author Lydia Buntrock <lydia.buntrock AT fu-berlin.de>
  */
 
 #pragma once
 
-#include <seqan3/core/algorithm/configuration.hpp>
+#include <variant>
+
 #include <seqan3/core/algorithm/pipeable_config_element.hpp>
-#include <seqan3/core/detail/pack_algorithm.hpp>
 #include <seqan3/search/configuration/detail.hpp>
 #include <seqan3/search/configuration/max_error_common.hpp>
 #include <seqan3/search/detail/search_common.hpp>
 
 namespace seqan3::search_cfg
 {
-/*!\brief A configuration element for the maximum number of errors across all error types (mismatches, insertions,
- *        deletions). This is an upper bound of errors independent from error numbers of specific error types.
+/*!\brief Configuration element that represents the number or rate of total errors.
  * \ingroup search_configuration
- * \details A mismatch corresponds to diverging bases between text and query for a certain position.
- *          An insertion corresponds to a base inserted into the query that does not occur in the text at the position,
- *          a deletion corresponds to a base deleted from the query sequence that does occur in the indexed text.
- *          Deletions at the beginning and at the end of the sequence are not considered during a search.
+ * \tparam    max_error_total                       Error type.
+ * \param[in] std::variant<error_count, error_rate> A variant of the error number or rate.
+ *
+ * \details This configuration element can be used to determine the number or rate of total hits witch are supported.
+ *          A mismatch corresponds to diverging bases between text and query for a certain position.
```

Comment:
```suggestion
 * \details This configuration element can be used to determine the number or rate of total errors that are supported.
```

The second line seemed to be wrong